### PR TITLE
[10.0] Migrate product_categ_image

### PR DESCRIPTION
**Pull request by alexis-via**
The v8 version of product_categ_image is available here https://github.com/OCA/product-attribute/pull/201

### CODE REVIEW

**Review by bguillot — APPROVED** [review]

**Review by LRovira-PlanetaTIC — APPROVED** [review]

File: product_categ_image/models/product_category.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = 'product.category'
+
+    image = fields.Binary(string='Image')
```
Comment (pedrobaeza) [inline product_categ_image/models/product_category.py:12]: Shouldn't this have `attachment=True` for a better DB space management?

Comment (alexis-via) [reply product_categ_image/models/product_category.py:12]: Probably. But this is already used in production, so maybe we could do that when migrating to v11.

Comment (pedrobaeza) [reply product_categ_image/models/product_category.py:12]: You can add a migration script right now if you want. There's a method in openupgradelib for performing the extract. Any way, at least put a TODO here.

Comment (FFernandez-PlanetaTIC) [reply product_categ_image/models/product_category.py:12]: :+1: to put a TODO

Please, could we merge? Thanks

**Comment by pedrobaeza** [comment]
@FFernandez-PlanetaTIC can you superseed this adding extra `attachment=True` attribute and fixing possible Travis lint checks? (I think they will go out when you rebase)

**Pull request closed without merging**